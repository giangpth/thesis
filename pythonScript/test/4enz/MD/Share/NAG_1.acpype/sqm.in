Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   6    C1      -22.2330       91.7750       -6.5930 
   6    C2      -23.3060       92.5060       -5.7620 
   6    C3      -24.6090       91.6760       -5.6850 
   6    C4      -25.1190       91.3390       -7.0930 
   6    C5      -23.9930       90.5770       -7.8190 
   6    C6      -24.4650       90.1760       -9.2240 
   6    C7      -22.7970       94.0790       -3.9020 
   6    C8      -22.3810       94.1880       -2.4650 
   7    N2      -22.8780       92.8300       -4.3980 
   8    O3      -25.6170       92.3270       -4.9320 
   8    O4      -26.3630       90.6290       -7.0750 
   8    O5      -22.7690       91.3360       -7.8460 
   8    O6      -23.3770       89.9240      -10.0800 
   8    O7      -23.0290       95.1140       -4.5260 
   1   H11      -21.4010       92.4550       -6.7790 
   1   H12      -21.8730       90.9110       -6.0340 
   1    H2      -23.5380       93.4430       -6.2690 
   1    H3      -24.3750       90.7370       -5.1840 
   1    H4      -25.2800       92.2780       -7.6220 
   1    H5      -23.8010       89.6610       -7.2610 
   1   H61      -25.0730       89.2750       -9.1500 
   1   H62      -25.0690       90.9810       -9.6420 
   1   HN2      -22.6270       92.0640       -3.7890 
   1   HO3      -25.2820       92.5300       -4.0560 
   1   HO4      -26.8600       90.8340       -7.8710 
   1   HO6      -23.7030       89.6760      -10.9480 
   1   H81      -22.3500       95.2380       -2.1740 
   1   H82      -21.3920       93.7470       -2.3390 
   1   H83      -23.0980       93.6580       -1.8380 

