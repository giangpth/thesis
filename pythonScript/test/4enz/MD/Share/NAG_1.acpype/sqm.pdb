ATOM      1  C1  NAG  1111     -22.442  91.735  -6.562  1.00  0.00           C
ATOM      2  C2  NAG  1111     -23.522  92.447  -5.713  1.00  0.00           C
ATOM      3  C3  NAG  1111     -24.785  91.551  -5.689  1.00  0.00           C
ATOM      4  C4  NAG  1111     -25.210  91.208  -7.130  1.00  0.00           C
ATOM      5  C5  NAG  1111     -24.037  90.585  -7.896  1.00  0.00           C
ATOM      6  C6  NAG  1111     -24.386  90.390  -9.373  1.00  0.00           C
ATOM      7  C7  NAG  1111     -22.738  94.055  -3.996  1.00  0.00           C
ATOM      8  C8  NAG  1111     -22.104  94.223  -2.637  1.00  0.00           C
ATOM      9  N2  NAG  1111     -23.054  92.760  -4.386  1.00  0.00           N
ATOM     10  O3  NAG  1111     -25.907  92.212  -5.126  1.00  0.00           O
ATOM     11  O4  NAG  1111     -26.254  90.251  -7.114  1.00  0.00           O
ATOM     12  O5  NAG  1111     -22.917  91.474  -7.871  1.00  0.00           O
ATOM     13  O6  NAG  1111     -23.359  89.705 -10.054  1.00  0.00           O
ATOM     14  O7  NAG  1111     -23.003  95.031  -4.721  1.00  0.00           O
ATOM     15  H11 NAG  1111     -21.557  92.406  -6.724  1.00  0.00           H
ATOM     16  H12 NAG  1111     -22.118  90.778  -6.073  1.00  0.00           H
ATOM     17  H2  NAG  1111     -23.798  93.417  -6.229  1.00  0.00           H
ATOM     18  H3  NAG  1111     -24.587  90.602  -5.116  1.00  0.00           H
ATOM     19  H4  NAG  1111     -25.558  92.147  -7.645  1.00  0.00           H
ATOM     20  H5  NAG  1111     -23.749  89.601  -7.432  1.00  0.00           H
ATOM     21  H61 NAG  1111     -25.293  89.732  -9.451  1.00  0.00           H
ATOM     22  H62 NAG  1111     -24.582  91.385  -9.855  1.00  0.00           H
ATOM     23  HN2 NAG  1111     -22.700  92.004  -3.849  1.00  0.00           H
ATOM     24  HO3 NAG  1111     -25.643  92.565  -4.269  1.00  0.00           H
ATOM     25  HO4 NAG  1111     -26.967  90.634  -6.584  1.00  0.00           H
ATOM     26  HO6 NAG  1111     -22.560  90.243  -9.976  1.00  0.00           H
ATOM     27  H81 NAG  1111     -21.948  95.314  -2.448  1.00  0.00           H
ATOM     28  H82 NAG  1111     -21.116  93.704  -2.599  1.00  0.00           H
ATOM     29  H83 NAG  1111     -22.765  93.808  -1.839  1.00  0.00           H
