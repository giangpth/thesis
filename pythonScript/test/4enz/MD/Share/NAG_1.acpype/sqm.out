            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 41

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       71
| QMMM:    diag iterations used for timing =       15
| QMMM:
| QMMM:              Internal diag routine = 0.027456 seconds
| QMMM:                 Dspev diag routine = 0.131201 seconds
| QMMM:                Dspevd diag routine = 0.060192 seconds
| QMMM:                Dspevx diag routine = 0.144118 seconds
| QMMM:                 Dsyev diag routine = 0.104518 seconds
| QMMM:                Dsyevd diag routine = 0.032007 seconds
| QMMM:                Dsyevr diag routine = 0.078477 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.011308 seconds
| QMMM:
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C      -22.2330   91.7750   -6.5930
  QMMM:     2        2      C      -23.3060   92.5060   -5.7620
  QMMM:     3        3      C      -24.6090   91.6760   -5.6850
  QMMM:     4        4      C      -25.1190   91.3390   -7.0930
  QMMM:     5        5      C      -23.9930   90.5770   -7.8190
  QMMM:     6        6      C      -24.4650   90.1760   -9.2240
  QMMM:     7        7      C      -22.7970   94.0790   -3.9020
  QMMM:     8        8      C      -22.3810   94.1880   -2.4650
  QMMM:     9        9      N      -22.8780   92.8300   -4.3980
  QMMM:    10       10      O      -25.6170   92.3270   -4.9320
  QMMM:    11       11      O      -26.3630   90.6290   -7.0750
  QMMM:    12       12      O      -22.7690   91.3360   -7.8460
  QMMM:    13       13      O      -23.3770   89.9240  -10.0800
  QMMM:    14       14      O      -23.0290   95.1140   -4.5260
  QMMM:    15       15      H      -21.4010   92.4550   -6.7790
  QMMM:    16       16      H      -21.8730   90.9110   -6.0340
  QMMM:    17       17      H      -23.5380   93.4430   -6.2690
  QMMM:    18       18      H      -24.3750   90.7370   -5.1840
  QMMM:    19       19      H      -25.2800   92.2780   -7.6220
  QMMM:    20       20      H      -23.8010   89.6610   -7.2610
  QMMM:    21       21      H      -25.0730   89.2750   -9.1500
  QMMM:    22       22      H      -25.0690   90.9810   -9.6420
  QMMM:    23       23      H      -22.6270   92.0640   -3.7890
  QMMM:    24       24      H      -25.2820   92.5300   -4.0560
  QMMM:    25       25      H      -26.8600   90.8340   -7.8710
  QMMM:    26       26      H      -23.7030   89.6760  -10.9480
  QMMM:    27       27      H      -22.3500   95.2380   -2.1740
  QMMM:    28       28      H      -21.3920   93.7470   -2.3390
  QMMM:    29       29      H      -23.0980   93.6580   -1.8380

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -234.6897 kcal/mol        3.7497 kcal/(mol*A)
xmin    20     -237.6653 kcal/mol        1.9934 kcal/(mol*A)
xmin    30     -239.3312 kcal/mol        0.2127 kcal/(mol*A)
xmin    40     -239.3737 kcal/mol        0.0734 kcal/(mol*A)
xmin    50     -239.3769 kcal/mol        0.0087 kcal/(mol*A)
xmin    60     -239.3770 kcal/mol        0.0015 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -41.5812  -39.8037  -38.7206  -37.7931  -37.2264  -35.6233  -32.1898  -31.7075
 -29.9441  -27.1550  -24.8227  -23.4354  -21.9403  -21.6841  -19.3540  -18.7759
 -18.3850  -17.9588  -17.3456  -17.1046  -16.6326  -16.3447  -16.1160  -15.8579
 -15.1619  -15.0687  -14.9106  -14.4840  -13.9471  -13.7617  -13.5887  -13.2772
 -12.9142  -12.6628  -12.1557  -11.6384  -11.4005  -11.1775  -11.1351  -10.9055
 -10.3112    0.9573    1.5436    2.4611    2.7985    2.9458    3.0703    3.2273
   3.2871    3.3960    3.5254    3.6406    3.6569    3.7193    3.8260    3.8747
   3.9348    4.1059    4.1352    4.2404    4.3512    4.3849    4.5775    4.7505
   4.9093    5.6721    5.7618    5.8949    6.1174    6.3998    6.6430

 Heat of formation   =       -239.37696305 kcal/mol  (      -10.38016405 eV)

 Total SCF energy    =     -70145.66132715 kcal/mol  (    -3041.74412762 eV)
 Electronic energy   =    -382391.31647282 kcal/mol  (   -16581.73177541 eV)
 Core-core repulsion =     312245.65514567 kcal/mol  (    13539.98764779 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.038
     2      C                 -0.048
     3      C                 -0.014
     4      C                  0.015
     5      C                 -0.016
     6      C                 -0.009
     7      C                  0.299
     8      C                 -0.244
     9      N                 -0.389
    10      O                 -0.332
    11      O                 -0.324
    12      O                 -0.270
    13      O                 -0.323
    14      O                 -0.350
    15      H                  0.120
    16      H                  0.070
    17      H                  0.157
    18      H                  0.086
    19      H                  0.105
    20      H                  0.107
    21      H                  0.130
    22      H                  0.077
    23      H                  0.217
    24      H                  0.217
    25      H                  0.224
    26      H                  0.209
    27      H                  0.122
    28      H                  0.100
    29      H                  0.103
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE     1.929   -0.552    4.628    5.044
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C      -22.4424   91.7350   -6.5616
  QMMM:     2        2      C      -23.5216   92.4465   -5.7134
  QMMM:     3        3      C      -24.7850   91.5513   -5.6887
  QMMM:     4        4      C      -25.2101   91.2079   -7.1302
  QMMM:     5        5      C      -24.0367   90.5847   -7.8955
  QMMM:     6        6      C      -24.3861   90.3902   -9.3735
  QMMM:     7        7      C      -22.7379   94.0545   -3.9956
  QMMM:     8        8      C      -22.1038   94.2234   -2.6367
  QMMM:     9        9      N      -23.0541   92.7595   -4.3857
  QMMM:    10       10      O      -25.9067   92.2116   -5.1262
  QMMM:    11       11      O      -26.2542   90.2505   -7.1139
  QMMM:    12       12      O      -22.9169   91.4737   -7.8714
  QMMM:    13       13      O      -23.3586   89.7049  -10.0542
  QMMM:    14       14      O      -23.0027   95.0305   -4.7214
  QMMM:    15       15      H      -21.5566   92.4057   -6.7238
  QMMM:    16       16      H      -22.1182   90.7780   -6.0726
  QMMM:    17       17      H      -23.7980   93.4174   -6.2288
  QMMM:    18       18      H      -24.5870   90.6021   -5.1162
  QMMM:    19       19      H      -25.5581   92.1467   -7.6450
  QMMM:    20       20      H      -23.7487   89.6014   -7.4322
  QMMM:    21       21      H      -25.2925   89.7317   -9.4513
  QMMM:    22       22      H      -24.5818   91.3845   -9.8549
  QMMM:    23       23      H      -22.7002   92.0045   -3.8487
  QMMM:    24       24      H      -25.6435   92.5651   -4.2688
  QMMM:    25       25      H      -26.9666   90.6336   -6.5837
  QMMM:    26       26      H      -22.5601   90.2428   -9.9759
  QMMM:    27       27      H      -21.9485   95.3139   -2.4481
  QMMM:    28       28      H      -21.1159   93.7044   -2.5995
  QMMM:    29       29      H      -22.7653   93.8081   -1.8388

           --------- Calculation Completed ----------

